API/Async.pm

```diff
@@ -168,6 +168,42 @@ sub featured {
 	$self->_get("/featured", $cb);
 }
 
+sub page {
+	my ($self, $cb, $path, $limit) = @_;
+	
+	$self->_get("/$path", sub {
+		my $page = shift;
+
+		my $items = [];		
+		# flatten down all modules as they seem to be only one per row					
+		push @$items, @{$_->{modules}} foreach (@{$page->{rows}});
+
+		$cb->($items || []);
+	}, { 
+		_ttl => DYNAMIC_TTL, 
+		deviceType => 'BROWSER',
+		limit => $limit || DEFAULT_LIMIT,
```

Comment:
The "pages" endpoints seem to accept a local parameter in addition to the country. I've now a mix of German/English/Norwegian, as the demo account they provided me with years ago is Norwegian (WiMP - before TIDAL!), my system is German, and this call returns English without the locale...

The `locale` value must be lower case:

```diff
diff --git a/API/Async.pm b/API/Async.pm
index 8799e76..f728bed 100644
--- a/API/Async.pm
+++ b/API/Async.pm
@@ -35,6 +35,7 @@ use constant CAN_MORE_HTTP_VERBS => Slim::Networking::SimpleAsyncHTTP->can('dele
 my $cache = Slim::Utils::Cache->new();
 my $log = logger('plugin.tidal');
 my $prefs = preferences('plugin.tidal');
+my $serverPrefs = preferences('server');
 
 my %apiClients;
 
@@ -183,6 +184,7 @@ sub page {
                _ttl => DYNAMIC_TTL,
                deviceType => 'BROWSER',
                limit => $limit || DEFAULT_LIMIT,
+               locale => lc($serverPrefs->get('language')),
        } );
 }
 
@@ -200,7 +202,7 @@ sub dataPage {
                _page => PLAYLIST_LIMIT,
                deviceType => 'BROWSER',
                limit => $limit || DEFAULT_LIMIT,
-               locale => preferences('server')->get('language'),
+               locale => lc($serverPrefs->get('language')),
        } );
 }
```